src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1062,6 +1068,13 @@ private void getFieldInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_MET
                     else
                     if (helperId != ReadyToRunHelperId.Invalid)
                     {
+                        if (_compilation.SymbolNodeFactory.VerifyTypeAndFieldLayout)
+                        if (_compilation.SymbolNodeFactory.VerifyTypeAndFieldLayout)
```

Comment:
copy-paste issue?

Reply:
Lol, yes. Conveniently, it doesn't have an impact on the correctness, but it is a bit ugly  :)

Reply:
wonder if the compiler optimizes it out :)